### PR TITLE
doc: fixed typo in doc mainpage.md

### PR DESCRIPTION
**Pull request by emmanuelsearch**
one slash too much

### CODE REVIEW

File: doc/doxygen/src/mainpage.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ call this the `native` port). Try it right now in your terminal window:
 git clone git://github.com/RIOT-OS/RIOT.git # assumption: git is pre-installed
 git checkout <LATEST_RELEASE>
 cd RIOT
-./dist/tools//tapsetup/tapsetup             # create virtual Ethernet
+./dist/tools/tapsetup/tapsetup             # create virtual Ethernet
                                             # interfaces to connect to RIOT
```
Comment (authmillenon) [inline doc/doxygen/src/mainpage.md]: Please fill the indentation of the comment too.

**Comment by emmanuelsearch** [comment]
Indentation fixed

**Comment by emmanuelsearch** [comment]
(and commits squashed)

**Comment by haukepetersen** [comment]
ACK and go

**Pull request merged**